influxdb3/src/commands/serve.rs

```diff
@@ -148,6 +148,11 @@ const DEPRECATED_ENV_VARS: &[(&str, &str)] = &[(
     "use INFLUXDB3_PARQUET_MEM_CACHE_SIZE instead, it is in MB or %",
 )];
 
+const MIN_REPLAY_PRELOAD_CONCURRENCY: usize = 10; // the min number of files that will be held in memory
+fn wal_replay_concurrency_limit_default() -> String {
```

Comment:
I didn't know the clap default value macro would take a function call but it does so this works nicely to dynamically determine the default value. thanks for the suggestion. 

But of course since the there's the "short help" via `-h` or `--help` that's from clap and then the "all help" from the curated help-all file, we can't actually get clap to _print_ in cli help the calculated value. 

but i tested it by commenting out the maybe_print_help() method and it works.